CLI/commands/permission_manager.js

```diff
@@ -239,6 +244,46 @@ async function getModulesWithPermissions() {
   return modules;
 }
 
+async function getDelegatesAndPermissions() {
+  let moduleABI = abis.moduleInterface();
+  let result = [];
+  for (const type in MODULES_TYPES) {
+    let modulesAttached = await securityToken.methods.getModulesByType(MODULES_TYPES[type]).call();
+    for (const module of modulesAttached) {
+      let contractTemp = new web3.eth.Contract(moduleABI, module);
+      let permissions = await contractTemp.methods.getPermissions().call();
+      if (permissions.length > 0) {
+        for (const permission of permissions) {
+          let allDelegates = await generalPermissionManager.methods.getAllDelegatesWithPerm(module, permission).call();
+          let moduleName = web3.utils.hexToUtf8((await securityToken.methods.getModule(module).call())[0]);
+          let permissionName = web3.utils.hexToUtf8(permission);
+          for (delegateAddr of allDelegates) {
+            if (result[delegateAddr] != undefined) {
+              result[delegateAddr].push({module: moduleName, permission: permissionName})
+            } else {
+              result[delegateAddr] = [{module: moduleName, permission: permissionName}]
+            }
+          }
+        }
+      }
+    }
+  }
+  return result
+}
+
+function renderTable(permisions, address) {
+  let result = ``;
+  if (permisions[address] != undefined) {
+    for (const obj of permisions[address]) {
+      result += `
+      ${obj.module} -> ${obj.permission}`;
```

Comment:
Instead of repeat each module for every permission granted, it should group all perms for each module in one line.

Reply:
makes sense!